static/app/utils/isActiveSuperuser.tsx

```diff
@@ -16,7 +16,7 @@ const SUPERUSER_COOKIE_DOMAIN = window.superUserCookieDomain;
  *
  * Documented here: https://getsentry.atlassian.net/browse/ER-1602
  */
-export function isActiveSuperuser(organization?: Organization) {
+export function isActiveSuperuser(organization?: Organization | null) {
```

Comment:
i have to fix getsentry first

Reply:
Why do we support both null and undefined here? It seems redundant?

Reply:
it will fail checks in getsentry because i am not passing in anything into `isActiveSuperuser` there yet. sometimes `organization` is null in components in sentry and getsentry

Reply:
@cathteng I think instead of allowing `undefined` and `null` we should stick with one, and either remove `undefined` or `null`. IMHO, removing `null` feels like the correct approach.

Reply:
i also forgot we have an `OrganizationStore` that we can fetch the organization from so i can remove the organization parameter entirely, but thank you